pkg/abstract/model/telemetry.go

```diff
@@ -0,0 +1,5 @@
+package model
+
+type Telemetry struct {
```

Comment:
Please add a configuration example into https://github.com/doublecloud/transfer/blob/main/docs/integrations/connect-prometheus-to-transfer.md and probably https://github.com/doublecloud/transfer/blob/main/docs/deploy_k8s.md

Reply:
i'm not sure that this is the best place for this kind of setting.
this is an infra level setting, so better to pass it either as separate cli-arge [here](https://github.com/doublecloud/transfer/blob/82562e8d61b2754f00e7b00454d170f1b95af7a7/cmd/trcli/main.go#L70) or maybe as extra optional config file in a same place (in cli args)

Reply:
Good catch @laskoviymishka